source/main/physics/Beam.cpp

```diff
@@ -1202,7 +1202,7 @@ void Actor::ResetPosition(Vector3 target)
 void Actor::HandleMouseMove(int node, Vector3 pos, float force)
 {
     m_mouse_grab_node = node;
-    m_mouse_grab_move_force = force;
+    m_mouse_grab_move_force = force * std::pow(m_total_mass / 3000.0f, 0.75f);

```

Comment:
Are you sure it's subtle? Looks like quite heavy scale-down to me.

Reply:
Y-Axis: Scale
X-Axis: Mass in kg
![temp](https://user-images.githubusercontent.com/9437207/52335702-54f32480-2a03-11e9-84c9-2535ff0613d8.png)

Reply:
My math is worse than I feared. 😆